book/running-wallaroo/wallaroo-command-line-options.md

```diff
@@ -65,6 +65,6 @@ argument:
 
 If you do not specify the number of `ponythreads`, the process will try to use all available cores.
 
-Since Machida (used for the Python API) is single-threaded, you must run it with `--ponythreads 1` or the cluster will crash.
+Since Machida (used for the Python API) is single-threaded, you must run it with `--ponythreads 1` or Machida will exit early.
```

Comment:
It seems to me that saying `Machida will exit early` is more ambiguous then saying `Machida will refuse to start`